src/getHumanAge.test.js

```diff
@@ -7,4 +7,22 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should give zero human years if animal < 15', () => {
+    const res = getHumanAge(14, 14);
+
+    expect(res).toEqual([0, 0]);
+  });
+
+  test('should give one human years if 15 <= animal >= 23', () => {
```

Comment:
The test description 'should give one human years if 15 <= animal >= 23' is misleading. The condition '15 <= animal >= 23' is not logically correct. It should be something like 'should give one human year if 15 <= animal < 24' to accurately describe the range being tested.